cli.js

```diff
@@ -44,6 +44,14 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   .command('create <function> [options]', 'Create a function from template', (yargs0) => {
     yargs0
       .usage('Usage: $0 create <function> [options]')
+      // TODO: should runtime be positional (required)?
+      .option('runtime', {
+        alias: 'r',
+        choices: ['node8', 'cpython2', 'pypy2'],
```

Comment:
`cpython2`-> `python2`

Not sure most python users known enough not to be confused by the term `cpython`. Many of them, I suspect, just know that interpreter as the "regular" python. And `python2` is what the other vendors call it as well.